backend/ops_api/ops/resources/agreements.py

```diff
@@ -243,6 +250,38 @@ def patch(self, id: int) -> Response:
             current_app.logger.error(f"{message_prefix}: {se}")
             return make_response_with_headers({}, 500)
 
+    @override
+    @jwt_required()
+    def delete(self, id: int) -> Response:
+        message_prefix = f"DELETE from {ENDPOINT_STRING}"
+
+        identity = get_jwt_identity()
+        is_authorized = self.auth_gateway.is_authorized(identity, ["PATCH_AGREEMENT"])
```

Comment:
I know it's mostly just a placeholder at this point, but shouldn't this be `DELETE_AGREEMENT` or something similar?

Reply:
Missed that. Oops. Fixed now.